c/datasetjson.c

```diff
@@ -1602,34 +1655,520 @@ void respondWithDataset(HttpResponse* response, char* absolutePath, int jsonMode
   );
 
   if (daRC != RC_DYNALLOC_OK) {
-    zowelog(NULL, LOG_COMP_DATASERVICE, ZOWE_LOG_DEBUG,
-    		    "error: ds alloc dsn=\'%44.44s\', member=\'%8.8s\', dd=\'%8.8s\',"
+    zowelog(NULL, LOG_COMP_DATASERVICE, ZOWE_LOG_SEVERE,
+            "error: ds alloc 5 dsn=\'%44.44s\', member=\'%8.8s\', dd=\'%8.8s\',"
             " rc=%d sysRC=%d, sysRSN=0x%08X (read)\n",
             daDsn.name, daMember.name, daDDname.name, daRC, daSysRC, daSysRSN);
     respondWithDYNALLOCError(response, daRC, daSysRC, daSysRSN,
                              &daDsn, &daMember, "r");
     return;
   }
 
-  zowelog(NULL, LOG_COMP_DATASERVICE, ZOWE_LOG_DEBUG,
+  zowelog(NULL, LOG_COMP_DATASERVICE, ZOWE_LOG_INFO,
           "debug: reading dsn=\'%44.44s\', member=\'%8.8s\', dd=\'%8.8s\'\n",
           daDsn.name, daMember.name, daDDname.name);
 
   DDName ddName;
   memcpy(&ddName.value, &daDDname.name, sizeof(ddName.value));
   respondWithDatasetInternal(response, absolutePath, &dsn, &ddName, jsonMode);
 
+zowelog(NULL, LOG_COMP_DATASERVICE, ZOWE_LOG_INFO,
+  "ZSS1778I %s wait around for manual check of ENQ ...\n", __FUNCTION__);
```

Comment:
Do not put message IDs, they are product specific. Again, this is common/library code.

Reply:
Yes, accepted.  I just used line numbers.  I never intended them to be merged in that form.